cli/account_command.go

```diff
@@ -136,6 +146,39 @@ func (c *actCmd) backupAction(_ *kingpin.ParseContext) error {
 	return nil
 }
 
+func (c *actCmd) restoreAction(kp *kingpin.ParseContext) error {
+	_, mgr, err := prepareHelper("", natsOpts()...)
+	kingpin.FatalIfError(err, "setup failed")
+	streams, err := mgr.StreamNames(nil)
+	if err != nil {
+		return err
+	}
+	existingStreams := map[string]struct{}{}
+	for _, n := range streams {
+		existingStreams[n] = struct{}{}
+	}
+	de, err := os.ReadDir(c.backupDirectory)
+	kingpin.FatalIfError(err, "setup failed")
+	for _, d := range de {
+		if !d.IsDir() {
+			kingpin.FatalIfError(err, "expected a directory")
+		}
+		if _, ok := existingStreams[d.Name()]; ok {
+			kingpin.Fatalf("stream %q exists already", d.Name())
```

Comment:
wondering if maybe we should support a replace mode? Otherwise LGTM

Reply:
I am checking these beforehand, because stream restore fails them. Basically want to avoid a partial restore.
So any change would have to start with stream restore.

Would that basically delete the stream before restoring it?

Reply:
Yes or prompts for replace/skip? With matching flags to say always skip or always replace without prompting for non interactive use. 

I wasn’t sure about this when I made the backup and Todd wanted a release but the choices are basically as above or what you have now as default with flags for always skip and always delete - ie. no prompts.

still not sure tbh

Reply:
Since you are not sure, I will skip that for now.